modules/nw-sriov-dpdk-example-mellanox.adoc.adoc

```diff
@@ -0,0 +1,146 @@
+  namespace: openshift-sriov-network-operator
+spec:
+  networkNamespace: <target_namespace>
+  ipam: |- <1>
+# ...
+  vlan: <vlan>
+  resourceName: mlxnics
+----
+<1> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
++
+[NOTE]
+=====
+See the "Configuring SR-IOV additional network" section for a detailed explanation on each option in `SriovNetwork`.
+=====
++
+An optional library, app-netutil, provides several API methods for gathering network information about a container's parent pod.
+
+. Create the `SriovNetworkNodePolicy` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f mlx-dpdk-network.yaml
+----
+
+. Create the following `Pod` spec, and then save the YAML in the `mlx-dpdk-pod.yaml` file.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  namespace: <target_namespace> <1>
+  annotations:
+    k8s.v1.cni.cncf.io/networks: mlx-dpdk-network
+spec:
+  containers:
+  - name: testpmd
+    image: <DPDK_image> <2>
+    securityContext:
+      runAsUser: 0
+      capabilities:
+        add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"] <3>
+    volumeMounts:
+    - mountPath: /dev/hugepages <4>
+      name: hugepage
+    resources:
+      limits:
+        openshift.io/mlxnics: "1" <5>
+        memory: "1Gi"
+        cpu: "4" <6>
+        hugepages-1Gi: "4Gi" <7>
+      requests:
+        openshift.io/mlxnics: "1"
+        memory: "1Gi"
+        cpu: "4"
+        hugepages-1Gi: "4Gi"
+    command: ["sleep", "infinity"]
+  volumes:
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+----
+<1> Specify the same `target_namespace` where `SriovNetwork` object `mlx-dpdk-network` is created. If you would like to create the pod in a different namespace, change `target_namespace` in both `Pod` spec and `SriovNetowrk` object.
+<2> Specify the DPDK image which includes your application and the DPDK library used by application.
+<3> Specify additional capabilities required by the application inside the container for hugepage allocation, system resource allocation, and network interface access.
+<4> Mount the hugepage volume to the DPDK pod under `/dev/hugepages`. The hugepage volume is backed by the emptyDir volume type with the medium being `Hugepages`.
+<5> Optional: Specify the number of DPDK devices allocated to the DPDK pod. This resource request and limit, if not explicitly specified, will be automatically added by SR-IOV network resource injector. The SR-IOV network resource injector is an admission controller component managed by SR-IOV Operator. It is enabled by default and can be disabled by setting the `enableInjector` option to `false` in the default `SriovOperatorConfig` CR.
+<6> Specify the number of CPUs. The DPDK pod usually requires exclusive CPUs be allocated from kubelet. This is achieved by setting CPU Manager policy to `static` and creating a pod with `Guaranteed` QoS.
```

Comment:
```suggestion
<6> Specify the number of CPUs. The DPDK pod usually requires that exclusive CPUs be allocated from kubelet. This is achieved by setting CPU Manager policy to `static` and creating a pod with `Guaranteed` QoS.
```

Reply:
Is there a reason to use passive voice here "This is achieved by setting CPU..." ?
Or could it be written as "To do this, set CPU..."